lib/src/widgets/platform_scaffold.dart

```diff
@@ -70,6 +70,7 @@ class PlatformAppBar extends StatelessWidget {
     this.leading,
     this.actions = const [],
     this.androidTitleSpacing,
+    this.bottom,
```

Comment:
Thanks for letting me know about this feature of CupertinoNavigationBar I wasn't aware of ;)

Now putting that in `PlatformScaffold` might lead to some issue. The way the platform scaffold wraps PlatformBottomBar in PreferredSize is not ideal, I will refactor this. Also I tested the `bottom` property of `CupertinoNavigationBar`, and it still has some issues with routes transition, currently being fixed in flutter repo.

I think PlatformScaffold should stay very limited, and don't support too much use cases. For now I'd say lets remove the `bottom` from platform scaffold and use separately `Scaffold` and `CupertinoPageScaffold` in your PR.